packages/edit-post/src/components/layout/index.js

```diff
@@ -124,12 +124,6 @@ function Layout( { styles } ) {
 			documentLabel: postTypeLabel || _x( 'Document', 'noun' ),
 		};
 	}, [] );
-	const [ distractionFree, setDistractionFree ] =
-		useState( isDistractionFree );
-
-	useEffect( () => {
-		setDistractionFree( isDistractionFree );
-	}, [ isDistractionFree ] );
```

Comment:
AFAIK, this was useless local state.

Reply:
These bits of code are surprising. I wouldn't have added it for no reason :D 🤷🏻 Probably didnt remove it on iteration X when it wasn't needed anymore 🙇🏻

Reply:
Yeah, sometimes it's hard to keep track of all the necessary/unnecessary bits as we iterate over feedback.